doc/classes/EditorPlugin.xml

```diff
@@ -214,6 +210,12 @@
 			<argument index="0" name="event" type="InputEvent">
 			</argument>
 			<description>
+				This method is called when there is an input event in the 2D viewport, e.g. the user clicks with the mouse in the 2D space (canvas item GUI). Keep in mind that for this method to be called you have to first declare the virtual method [method handles] so the editor knows that you want to work with the workspace:
```

Comment:
`Called when there is an input event in the 2D viewport. Works similarly to [method Control._gui_input]. You must define a [method handles] for this input callback to work.`

I'd move/leave the code example to the handles method.